contracts/test/accounts/GlobalAdmin.sol

```diff
@@ -1,17 +1,25 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.6.11;
 
-import { IMapleGlobals } from "../../../modules/globals/contracts/interfaces/IMapleGlobals.sol"; 
+interface IMapleGlobalsLike {
+    function setProtocolPause(bool) external;
+}
 
 contract GlobalAdmin {
 
+    /************************/
+    /*** Direct Functions ***/
+    /************************/
+    function setProtocolPause(address globals, bool pause) external {
```

Comment:
If And when GlobalAdmin is inherited, this function will not have a contract domain and may collide. Why is this account needed at all, especially when it's not part of what is tested by the unit tests? Can't the test (`this`) he the global admin of the mock admin just to set the protocol paused? Further, if there is a mock globals, then surely it can be made such that any account can call pause.